decks/ansible-networking.html

```diff
@@ -0,0 +1,1081 @@
+          <ul>
+            <li>What is Ansible and The Ansible Way</li>
+            <li>Installing Ansible</li>
+            <li>How Ansible Works and its Key Components</li>
+            <li>Ad-Hoc Commands</li>
+            <li>Playbook Basics</li>
+            <li>Reuse and Redistribution of Ansible Content with Roles</li>
+          </ul>
+          <aside class="notes">
+            <p>This deck is designed to provide students with direct introductory instruction and guidance to beginning to automate with Ansible. It is the starting point for students intent on becoming more proficient with Ansible through other Lightbulb modules and their own usage.</p>
+            <p>This deck supports lecture and hands-on forms of presenting this material. </p>
+            <p>Allow 2 hours to deliver the lecture-based form and 4 hours for stopping to do the workshop assignments. To access the additional slides for delivering the workshops, navigate down when available. </p>
+            <p>See the <a href="../facilitator/README.md">Ansible Lighttbulb facilitator&rsquo;s guide</a> for more details on using this deck and it&rsquo;s associated material.</p>
+          </aside>
+        </section>
+        <section>
+          <h2>What is Ansible?</h2>
+          <p>It&apos;s a <b>simple automation language</b> that can perfectly describe an IT application infrastructure in Ansible Playbooks.</p>
+          <p>It&apos;s an <b>automation engine</b> that runs Ansible Playbooks.</p>
+          <p>Ansible Tower is an <b>enterprise framework</b> for controlling, securing  and managing your Ansible automation with a <b>UI and RESTful API</b>.</p>
+          <aside class="notes speaker">
+            <p>Ansible is an automation platform:
+              <ul>
+                <li>Playbooks make up the automation language</li>
+                <li>The code base is the automation engine.</li>
+                <li>Ansible Tower manages existing automation</li>
+              </ul>
+            </p>
+          </aside>
+        </section>
+        <section>
+            <h2>Ansible Is...</h2>
+            <img src="images/simple-powerful-agentless-diagram.png" />
+            <aside class="notes">
+              <p>Ansible has a number of qualities that make it the most rapidly growing automation platform in the world. </p>
+              <p><strong>Ansible is simple.</strong> Playbooks are human and machine readable, no special coding skills required &ndash; and even people in your IT organization that don’t know Ansible can read an Ansible playbook and understand what’s happening.</p>
+              <p>This simplicity also means that it&rsquo;s easy to install and get startedto do real work with it quickly &ndash; usually in just minutes. </p>
```

Comment:
startedto -> started to

Reply:
fixed with commit: https://github.com/network-automation/lightbulb/commit/a3b9e87f51ffd086da0c38e138a33f44e149e47a